model.js

```diff
@@ -106,19 +91,16 @@ export class Game {
   convertToRubble() {
    this.rubble = this.rubble.concat(this.fallingPiece.points())
    this.startAPiece()
- }
- startAPiece() {
-   this.fallingPiece = new Piece(shapes.selectRandom(), this.rows, this.cols);
- }
- rotate() {
-   this.transactionDo(
-     () => this.fallingPiece.rotate(),
-     () => this.fallingPiece.unRotate())
-     return this
- }
- // unRotate() {
- //   this.rotation = Piece.rotations()[(Piece.rotations.indexOf(this.rotation)-1) % 4]
- // }
+   }
+   startAPiece() {
+     this.fallingPiece = new Piece(shapes.selectRandom(), this.rows, this.cols);
+   }
+   rotate() {
+     this.transactionDo(
```

Comment:
Rename `transactionDo` and the `thing` parameter to identifiers that speak more to their functionality.